highfidelity/hifi
# Perspective bug

This pr fixes the bug 11293 in Manuscript.
https://highfidelity.fogbugz.com/f/cases/11293/When-entering-VR-mode-with-a-short-avatar-perspective-is-incorrect

The fix is to call rigready when we change modes vr/desktop.  This will reset the height of the model to be whatever the current. This resets the scale when you enter vr mode.  

### Test Plan
1.  Open interface and go to the market place and choose the robimo avatar.
2.  open the tablet and select 'exit vr'
3.  go back into vr mode 
4. Expected result: the avatar should have the same perspective that you had last time you were in vr mode.
